static/app/views/settings/account/accountClose.tsx

```diff
@@ -147,16 +150,19 @@ function AccountClose() {
       </Alert>
 
       <Panel>
-        <PanelHeader>{t('Remove the following organizations')}</PanelHeader>
+        <PanelHeader>{t('Delete the following organizations')}</PanelHeader>
         <PanelBody>
           <PanelAlert type="info">
+            {tct('[strong:ORGANIZATIONS WITH CHECKED BOXES WILL BE DELETED]', {
+              strong: <strong />,
+            })}
+            <br />
             {t(
               'Ownership will remain with other organization owners if an organization is not deleted.'
             )}
             <br />
-            {tct(
-              "Boxes which can't be unchecked mean that you are the only organization owner and the organization [strong:will be deleted].",
-              {strong: <strong />}
+            {t(
+              "Boxes which can't be unchecked mean that you are the only organization admin and will be deleted with your account."
```

Comment:
"and will be deleted with your account" - this reads a little less clearly than it was before

Reply:
Yeah I honestly don't know if it's worth it to keep this line. I feel like the first line gets the point across, will be people be confused if they can't uncheck disabled boxes?